cudamat/cudamat.cu

```diff
@@ -28,6 +28,10 @@ EXPORT const char* get_last_cuda_error() {
     return cudaGetErrorString( err);
 }
 
+EXPORT const char* get_last_clib_error() {
+    return strerror(errno);
```

Comment:
Done in d591428 (amended). 

Apparently, Visual C's `stdlib.h` declares `errno` and includes `strerror`. I suppose it is a non-standard behaviour.